src/client.ts

```diff
@@ -11,16 +11,21 @@ export default class Client {
   public sendEmail(
     templateId: string,
     emailAddress: string,
-    options?: any
+    options?: any,
+    statusCallbackUrl?: string,
+    statusCallbackBearerToken?: string
   ): any {
     options = options || {};
-    const err = checkOptionsKeys(
+    statusCallbackUrl = statusCallbackUrl || "";
+    statusCallbackBearerToken = statusCallbackBearerToken || "";
```

Comment:
should these go under `options`—what was the reason to make them separate parameters to this function?

Reply:
I thought options are related to template & email/ sms mandatory params. 
And didn't think that its to categorize all optional values. So I added seperately.

Reply:
yeah they should probably just go under `options`. they are just as equal as the other generic options like `personalisation`, `reference`, `smsSenderId`. @dagoof thoughts?

Reply:
having additional parameters after the generic `options` looks odd to me too

Reply:
put it under options. it's not any different to the other parameters

Reply:
is there a reason we default to empty string? I would expect us to just not include the parameter if not present

Reply:
yeah that seems sensible to me to exclude if empty. and yeah @acodegirl let's go with putting these under `options`